opencti-platform/opencti-front/src/components/ItemIcon.jsx

```diff
@@ -519,7 +524,7 @@ const iconSelector = (type, variant, fontSize, color, isReversed) => {
     case 'default':
       return <CircleOutlined style={style} fontSize={fontSize} role="img" />;
     default:
-      return <HelpOutlined style={style} fontSize={fontSize} role="img" />;
+      return <AutoAwesomeMotion style={style} fontSize={fontSize} role="img" />;
```

Comment:
Rollback this icon to the previous one, we need to keep the "?" icon for the ones unknown.